nrf_modem/doc/gnss_interface.rst

```diff
@@ -505,7 +508,18 @@ Thus, |delta| t is always positive.
 A-GPS data
 **********
 
-GNSS automatically requests A-GPS data when GNSS is started for the first time or it determines that the existing data is going to go stale soon.
+You can use GNSS assistance data to shorten TTFF and decrease GNSS power consumption.
+See :ref:`lib_nrf_cloud_agps` and :ref:`lib_nrf_cloud_pgps` for information how to obtain assistance data from :ref:`lib_nrf_cloud` to be used with the nRF9160 SiP.
+
+Determining the assistance data needed by GNSS
+==============================================
+
+The GNSS interface has two methods for getting the current GNSS assistance data need.
+
+A-GPS data need event
+---------------------
+
+GNSS requests A-GPS data when GNSS is started for the first time or it determines that the existing data will soon be stale.
```

Comment:
I'd probably use `outdated` instead of `stale`. I guess `stale` is correct also because @peknis let it be there. My dictionary says it's like outdated but for food. You can decide whether to change or keep it.

Reply:
We can change the wording here, I'm not sure why `stale` was selected in the first place. We could also say `that the existing data will expire soon`. @peknis?

Reply:
Fine with your suggestion, @tokangas.

Reply:
Updated.